boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -284,7 +280,7 @@ class FsmStateMachineTest : public BaseStateMachineTest
         uint8_t   true_choice,
         uint8_t   false_choice)
     {
-        for (size_t pedal_percentage = 10; pedal_percentage <= 100;
+        for (size_t pedal_percentage = 11; pedal_percentage <= 100;
```

Comment:
lets try to get rid of these magic numbers if possible, can u do this for the current PR?